Resources/Textures/_Harmony/Objects/Devices/brigmedic_radio.rsi/meta.json

```
@@ -0,0 +1,14 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from commit: https://github.com/space-wizards/space-station-14/pull/4870/commits/d615a45615d4f0f0744bf06a5d46a1b3a5b4a424, and resprited by BoskiYourk for use in Harmony",
+    "size": {
+    "x": 32,
+    "y": 32
+    },
+    "states": [
+    {
+        "name": "icon"
+    }
+    ]
+}
```

Looks like this is missing a newline, has 4 space indentation and is overly flattened.
```suggestion
{
  "version": 1,
  "license": "CC-BY-SA-3.0",
  "copyright": "Taken from commit: https://github.com/space-wizards/space-station-14/pull/4870/commits/d615a45615d4f0f0744bf06a5d46a1b3a5b4a424, and resprited by BoskiYourk for use in Harmony",
  "size": {
    "x": 32,
    "y": 32
  },
  "states": [
    {
        "name": "icon"
    }
  ]
}

```